types/video.js/index.d.ts

```diff
@@ -5702,6 +5703,17 @@ declare namespace videojs {
         new (tracks?: Track[]): TrackList;
     };
 
+    interface UserActions {
+        doubleClick?: boolean | ((event: EventTarget.Event) => void);
```

Comment:
docs say that dblclick/keydown events are passed through to handlers, probably makes more sense to use MouseEvent/KeyboardEvent rather than EventTarget.Event if these are native events.

Reply:
Okay, I was pretty sure that it might sense to use `EventTarget.Event` because it is used everywhere else where I would expect otherwise, e.g.:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/4236a21f0f33d9cff7c88b9347b07235477d5889/types/video.js/index.d.ts#L654

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/4236a21f0f33d9cff7c88b9347b07235477d5889/types/video.js/index.d.ts#L665

Reply:
that's true. properly typing the events is probably a bigger discussion. for now, let's delete the `which: number` and leave it just `[key: string]: any;`. adding `which: number` implies that all events will have a non-null, numeric `which` property which isn't true. if you're feeling ambitious, maybe you can subclass `EventTarget.Event` as `KeyboardEvent` and add `which` to that?

Reply:
> if you're feeling ambitious, maybe you can subclass `EventTarget.Event` as `KeyboardEvent` and add `which` to that?

Will do. I guess `which?: number` would be the preferred option then?

Reply:
in a `KeyboardEvent` subclass I would define it as `readonly which: number;` which is how the core dom lib types it

Reply:
I did it for `hotkeys` but wasn't sure about `doubleClick` since `KeyboardEvent` doesn't make sense there and I kind of got lost in the documentation, sorry. What do you think?